tool/tctl/common/status_command.go

```diff
@@ -160,5 +163,29 @@ func (c *StatusCommand) Status(ctx context.Context, client auth.ClientI) error {
 		}
 		fmt.Print(view())
 	}
+
+	// Grab on login alerts.
+	alertCtx, _ := context.WithTimeout(ctx, constants.TimeoutGetClusterAlerts)
```

Comment:
This block of code is almost entirely duplicated in `tctl.go` and `tsh.go` can you abstract it out into a shared fn?